src/components/dashboard/FaceVerification/api/FaceVerificationApi.js

```diff
@@ -152,7 +152,7 @@ class FaceVerificationApi {
       // don't forget to set success = false flag
       // it's supposed that GoodServer will return false in the case of non-200 code
       // but let's add this additional safety check
-      response = { ...errorObject, success: false }
+      response = { ...failedResponse, success: false }
```

Comment:
yes, i did a mistake here.
also please, add some default message in the  line 162

e.g. const exception = new Error(error || 'An unexpected issue during the face verification API call')

Reply:
done